apisix/plugins/grpc-transcode/proto.lua

```diff
@@ -43,7 +43,7 @@ local function create_proto_obj(proto_id)
     end
 
     local _p  = protoc.new()
-    local res = _p:load(content)
+    local res = _p:load(content, "filename for loaded") -- https://github.com/starwing/lua-protobuf/commit/89faac205585360d800472c62752d5b532d7ce9e
```

Comment:
We need to explain it clearly in the comment instead of just a commit link.

Reply:
how about the explanation now?